electron/main/index.ts

```diff
@@ -12,11 +12,11 @@ import { Eos, Teos } from '@bitgo/sdk-coin-eos';
 import { Erc20Token, Eth, Gteth } from '@bitgo/sdk-coin-eth';
 import { Ethw } from '@bitgo/sdk-coin-ethw';
 import { Ltc } from '@bitgo/sdk-coin-ltc';
+import { Near, TNear } from '@bitgo/sdk-coin-near';
```

Comment:
it doesn't look like you're using `TNear` in the application code, is that intentional? if so, remove `TNear` here